### DISCUSSION TITLE
Add HS256 cert

### DISCUSSION THREAD

**Discussion by magiskboy** [casdoor/casdoor#2748] [Q&A] [answered]
I tried to create a new cert with HS256 algorithm but I got an error "Failed to save: crypto/rsa: message too long for RSA key size". In this picture, it is my configuration. How can I create a cert with HS256?

![image](https://github.com/casdoor/casdoor/assets/13352088/2ea0b298-51c1-4d86-abad-0f7f02394bef)

**Comment by hsluoyz** [comment, marked as answer]
@magiskboy issue created here: https://github.com/casdoor/casdoor/issues/2751

**Reply by hsluoyz** [reply]
Resolved by: https://github.com/casdoor/casdoor/pull/2793
